get-help.md

```diff
@@ -43,15 +43,13 @@ expect a first response within a few business hours.
 
 **Cancellations will be announced via email and on this webpage.**
 > 
->  Office hours are cancelled on **Thursday, April 23**  
+>  Due to HTCondor week, office hours are cancelled on **Tuesday, May 19th**  
 >
 > **Temporary Changes to Office Hours**
->
-> In light of [measures being taken by campus regarding COVID-19](https://covid19.wisc.edu/), 
-> we are replacing in-person office
-> hours with virtual office hours, and slightly decreasing our office 
-> hour availability. Starting on _Thursday, March 12_, office hours are: 
 > 
+> In light of [measures being taken by campus regarding COVID-19](https://covid19.wisc.edu/), 
```

Comment:
I like this change. Can you move lines 50 - 62 out of the blockquote? Now that it's not a new announcement, I think  I'd like it as regular text on the page.

Reply:
![image](https://user-images.githubusercontent.com/8378883/82239943-30247700-98ff-11ea-9e1c-cd9d74dc994d.png)

This is how the page looks now